nav2_bringup/bringup/rviz/nav2_default_view.rviz

```diff
@@ -5,12 +5,11 @@ Panels:
     Property Tree Widget:
       Expanded:
         - /Global Options1
-        - /RobotModel1/Status1
         - /TF1/Frames1
         - /TF1/Tree1
         - /Global Planner1/Global Costmap1
       Splitter Ratio: 0.5833333134651184
-    Tree Height: 464
+    Tree Height: 452
```

Comment:
Are the changes to this file needed?

Reply:
Yes, I should not have committed this file.